infra-templates/longhorn/0/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: '2'
+services:
+  longhorn-driver:
+    privileged: true
+    pid: host
+    image: imikushin/storage-longhorn:dev45
```

Comment:
There's no rancher image version for it, yet. The PR is in there: rancher/storage#27